container/container.go

```diff
@@ -1239,7 +1266,25 @@ func (b *ContainerWorker) start() {
 					glog.Infof("Receved configure command for agreement %v. Ignoring it because this agreement has been terminated.", agreementId)
 				} else if ags[0].AgreementExecutionStartTime != 0 {
 					glog.Infof("Receved configure command for agreement %v. Ignoring it because the containers for this agreement has been configured.", agreementId)
+				} else if ms_containers, err := b.findMicroserviceContainers(cmd.AgreementLaunchContext.Microservices); err != nil {
```

Comment:
The main purpose here is to find the network ids for the microservice containers.